llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -1953,8 +2006,8 @@ class VPWidenPointerInductionRecipe : public VPWidenInductionRecipe,
 /// recipe is placed in an entry block to a (non-replicate) region, it must have
 /// exactly 2 incoming values, the first from the predecessor of the region and
 /// the second from the exiting block of the region.
-class VPWidenPHIRecipe : public VPSingleDefRecipe {
-  /// Name to use for the generated IR instruction for the widened phi.
```

Comment:
Was this comment meant to be removed? No strong opinion if was intentional, I think the variable name is descriptive enough

Reply:
Accidentially dropped during merge, restored, thanks